src/index.html

```diff
@@ -1,14 +1,53 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
```suggestion
<html lang="en">
  <head>
```
Don't add empty lines between parent and child elements
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/a7715447-b952-494f-ab23-5e9532df697b)